install/common-pre-install/hpcm_installation-cpi.md

```diff
@@ -0,0 +1,806 @@
+            internal_name=service11, hostname1=node11, mgmt_bmc_net_macs=b4:2e:99:3b:70:f0, mgmt_net_macs=b8:59:9f:34:88:ea, mgmt_net_interfaces="enp65s0f0np0", mgmt_bmc_net_name=head-bmc, mgmt_net_name=head, redundant_mgmt_network=yes, switch_mgmt_network=yes, dhcp_bootfile=grub2, conserver_logging=yes, conserver_ondemand=no, root_type=disk, console_device=ttyS0, tpm_boot=no, mgmt_net_bonding_master=bond0, disk_bootloader=no, mgmtsw=mgmtsw0, predictable_net_names=yes, transport=udpcast, baud_rate=115200, bmc_username=root, bmc_password=initial0
+            internal_name=service12, hostname1=node12, mgmt_bmc_net_macs=b4:2e:99:3b:70:10, mgmt_net_macs=b8:59:9f:1d:d8:c2, mgmt_net_interfaces="enp66s0f0np0", mgmt_bmc_net_name=head-bmc, mgmt_net_name=head, redundant_mgmt_network=yes, switch_mgmt_network=yes, dhcp_bootfile=grub2, conserver_logging=yes, conserver_ondemand=no, root_type=disk, console_device=ttyS0, tpm_boot=no, mgmt_net_bonding_master=bond0, disk_bootloader=no, mgmtsw=mgmtsw0, predictable_net_names=yes, transport=udpcast, baud_rate=115200, bmc_username=root, bmc_password=initial0
+            ```
+
+         1. Add discovered nodes, set image for the discovered nodes.
+
+            ```
+            cm node discover add nodes.conf
+            cinstallman --assign-image --image sles15sp4 --kernel <kernel version> --node '*'
+            cinstallman --set-rootfs tmpfs --node '*'
+            cm node refresh netboot -n '*'
+            ```
+
+            __NOTE:__ Kernel version can be obtained from `cinstallman --show-default-image` command.
+
+         1. Stop the auto-discovery process and power on all nodes.
+
+            ```
+            cm node discover disable
+            cm power status -n '*'
+            cm power on -n '*'
+            ```
+            __NOTE:__ Power on the nodes when 'cm power status -n '*'' lists all the nodes. if there is any errors rerun the command
+            1. Check the status of nodes.
+
+            ```
+            cm power status -n '*'
+            ```
+
+            Example output (expected):
+
+            ```
+            node1        : BOOTED
+            node2        : BOOTED
+            node3        : BOOTED
+            node4        : BOOTED
+            node5        : BOOTED
+            node6        : BOOTED
+            node7        : BOOTED
+            node8        : BOOTED
+            node9        : BOOTED
+            node10       : BOOTED
+            node11       : BOOTED
+            node12       : BOOTED
+            ```
+
+            __NOTE:__ We need to wait at this step till all nodes report status as "BOOTED". Now the ports to all the NCNs can be enabled on switches.
+         1. Create a Bond Network
+	
+## Seed File generation
+
+   Here's the stepwise procedure to generate seedfiles.
+
+   1. Obtain `xnames.csv` file for the targetted system's installation.

```

Comment:
Instead of xnames.csv, change it to paddle file
will provide the proper input to this

Reply:
Addressed in https://github.com/abhishek-g-suresh/docs-csm/pull/5